BrushedMotorControllerFirmware/Kconfigs/Kconfig.speed_cntrl

```
@@ -1,11 +1,19 @@
 menu "Speed Control Configuration"
 
+config SPEED_CONTROL_ENABLE
+	bool "Enable functions responsible for speed control, like mode speed, \
+	     speed set, etc.. Continuus calculation of speed and speed read \
+	     is always enabled."
```

```suggestion
	bool "Enable functions responsible for speed control, like mode speed, \
	     speed set, etc."
```

imo there is no point to explain that continuous speed calculation is always enabled. dependant configs are descriptive well enough, and speed read will be visible from shell